zExceptions/ExceptionFormatter.py,sha256=Lwm_HGz7_Zam0o1Z4_os7ZKWCncnmtV6uqbsEzX6jnc,7831
zExceptions/ITracebackSupplement.py,sha256=OIXWg_uqq3WKMHj8lbQh5ebfXrl9sx4W2x0n9lKxh7o,2684
zExceptions/TracebackSupplement.py,sha256=B1dwNmhKuoVQaUtVUUxN797n3B3nzgYen1PZZ39sUDA,1399
zExceptions/__init__.py,sha256=oKYAkgovGBTwSsdCXFRkQsmwNbunKJLes6WH_Lnwdt0,11787
zExceptions/_compat.py,sha256=DO0NF5FZ8LtVXt2GJ96ReO2jTacA9hLYmZkY_POqubM,411
zExceptions/unauthorized.py,sha256=jkERyGzTnVb8EYjAx4vXolyyu2DdMqOlmyO8HflsDno,3599
zExceptions/tests/__init__.py,sha256=8pmNzrf_R6gngtAD-U4ztr27Y8n-S-ZHcarb_O4jnXE,36
zExceptions/tests/testExceptionFormatter.py,sha256=TwYRqIHmVohgOgqgzfCmUF4pMHTWOub490A9XUR4ZqU,4099
zExceptions/tests/test___init__.py,sha256=NYxbjMTxL-SLiFWaZQjj5utJmsAImFSEiQfSLi94JJ8,6867
zExceptions/tests/test_unauthorized.py,sha256=IUa-nM2Nmd5AvYwerSxaVHIpWEyfGLyA5Bgstw-c4qk,4355
zExceptions-4.3.dist-info/LICENSE.txt,sha256=PmcdsR32h1FswdtbPWXkqjg-rKPCDOo_r1Og9zNdCjw,2070
zExceptions-4.3.dist-info/METADATA,sha256=-vVbAXhN2LGismuhk0MHQI5yBu7VHWY7ahn322cJ3Fg,3673
zExceptions-4.3.dist-info/WHEEL,sha256=bb2Ot9scclHKMOLDEHY6B2sicWOgugjFKaJsT7vwMQo,110
zExceptions-4.3.dist-info/top_level.txt,sha256=zFCStwVrQuFYaKwnNCWmrJub2T8Yx4GRaQULvRThJrg,12
zExceptions-4.3.dist-info/RECORD,,
